profiles/forms.py

```diff
@@ -0,0 +1,21 @@
+
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
```

Comment:
make a story in the backlog to make password requirements more flexible and lenient.